pkg/clusters/addons/metallb/metallb.go

```diff
@@ -400,3 +404,33 @@ func metallbDeleteHack(ctx context.Context, kubeconfig *os.File) error {
 		WithStdout(io.Discard).
 		Do(ctx)
 }
+
+// -----------------------------------------------------------------------------
+// Metallb Builder
+// -----------------------------------------------------------------------------
+
+// Builder is a configuration tool for metallb cluster.Addons.
+type Builder struct {
```

Comment:
As far as I can see, addon builder are in almost all cases in their own files ([cert manager seems to be the only exception](https://github.com/Kong/kubernetes-testing-framework/blob/3fd788520f8608e49d2754f0269080091344c713/pkg/clusters/addons/certmanager/addon.go#L23-L44)).

I'd try to retain that and be consistent.